### DISCUSSION TITLE
How to prevent focus for any field with error after submit

### DISCUSSION THREAD

**Discussion by adammo94** [react-hook-form/react-hook-form#8419] [Q&A]
Hi, so I couldn't manage to find the way to do this.
I want to achieve something like this:
I have a form with validation (for some fields if that matters).
Right now if I just submit the form, focus is set to the first field that failed validation.

Can I prevent that somehow? After submitting I want to have all fields blurred when there are errors.

**Comment by bluebill1049** [comment]
`shouldFocusError: false` 
![Screen Shot 2022-05-30 at 9 37 25 pm](https://user-images.githubusercontent.com/10513364/170984448-6068060f-4f97-4e69-8bf9-94b9b8186f3d.png)
https://react-hook-form.com/api/useform
